execute/tokendata/usdc/attestation.go

```diff
@@ -4,22 +4,37 @@ import (
 	"context"
 	"errors"
 
+	"github.com/smartcontractkit/chainlink-common/pkg/hashutil"
 	cciptypes "github.com/smartcontractkit/chainlink-common/pkg/types/ccipocr3"
 
 	"github.com/smartcontractkit/chainlink-ccip/execute/exectypes"
 	"github.com/smartcontractkit/chainlink-ccip/pkg/reader"
+	"github.com/smartcontractkit/chainlink-ccip/pluginconfig"
 )
 
 var (
-	ErrDataMissing = errors.New("token data missing")
-	ErrNotReady    = errors.New("token data not ready")
-	ErrRateLimit   = errors.New("token data API is being rate limited")
-	ErrTimeout     = errors.New("token data API timed out")
+	ErrDataMissing     = errors.New("token data missing")
+	ErrNotReady        = errors.New("token data not ready")
+	ErrRateLimit       = errors.New("token data API is being rate limited")
+	ErrTimeout         = errors.New("token data API timed out")
+	ErrUnknownResponse = errors.New("unexpected response from attestation API")
 )
 
 type AttestationStatus struct {
+	// MessageHash is the hash of the message that the attestation was fetched for. It's going to be MessageSent event hash
+	MessageHash []byte
```

Comment:
You can replace `[]byte` with `cciptypes.Bytes`, that way you also get cool hex encoding out of the box in logs.